src/test/unit/controllers/SubscriptionManagementController.test.ts

```diff
@@ -106,7 +114,8 @@ describe('Subscription Management Controller', () => {
     const responseMock = sinon.mock(response);
     responseMock.expects('render').once().withArgs('error', request.i18n.getDataByLanguage(request.lng).error);
 
-    subscriptionManagementController.get(request, response);
-    responseMock.verify();
+    subscriptionManagementController.get(request, response).then(() => {
+      responseMock.verify();
+    });
   });
 });
```

Comment:
depending on if you choose to edit the controller or not, youll need to add additional tests

Reply:
added test